ts_scripts/install_dependencies.py

```diff
@@ -50,18 +50,21 @@ def install_python_packages(self, cuda_version, requirements_file_path, nightly)
             # as it may reinstall the packages with different versions
             os.system("conda install -y conda-build")
 
-        self.install_torch_packages(cuda_version)
         os.system(f"{sys.executable} -m pip install -U pip setuptools")
         # developer.txt also installs packages from common.txt
         os.system(f"{sys.executable} -m pip install -U -r {requirements_file_path}")
-        # If conda is available install conda-build package
 
-        # TODO: This will run 2 installations for torch but to make this cleaner we should first refactor all of our requirements.txt into just 2 files
-        # And then make torch an optional dependency for the common.txt
+        # Install dependencies for GPU
+        if cuda_version:
+            os.system(f"{sys.executable} -m pip install -U -r gpu.txt")
+
+        # Install PyTorch packages
         if nightly:
             os.system(
                 f"pip3 install numpy --pre torch[dynamo] torchvision torchtext torchaudio --force-reinstall --extra-index-url https://download.pytorch.org/whl/nightly/{cuda_version}"
```

Comment:
I thought I fixed this but can you remove the [dynamo] since you're here it's no longer needed

Reply:
Done